tests/kubeless/kubeless-tests.go

```diff
@@ -4,12 +4,14 @@ import (
 	"bytes"
 	"fmt"
 	"io/ioutil"
-	"log"
 	"math/rand"
 	"net/http"
 	"os/exec"
 	"strings"
 	"time"
+
+	log "github.com/sirupsen/logrus"
```

Comment:
Is switching to a new log library really justified considering the original scope?

Reply:
during the tests i had to modify logging. otherwise i was not able to see the reason for the error.

Reply:
Probably because the Logger was [not initialized with an appropriate Writer](https://golang.org/pkg/log/#New) ([stderr by default](https://golang.org/src/log/log.go?s=9437:9465#L73)). No big deal for me, I just wanted to point that out.

Reply:
nope, not because of that... it was just not giving enough information. But in order to not confuse people i removed those additional logs. And while i was at it, i quickly switched to logrus as it simplified implementing my needs